lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -77,32 +80,113 @@ def __init__(
         self._grpc_client_manager = grpc_client_manager
 
     async def _run(self) -> None:
-        subscribers = await self._get_subscribers()
-        if subscribers is not None:
-            self._process_subscribers(subscribers)
+        in_sync = await self._check_subscribers_in_sync()
+        if in_sync:
+            return
+
+        resync = await self._sync_subscribers()
+        if not resync:
+            return
+
+        ret = await self._get_all_subscribers()
+        if ret is None:
```

Comment:
Nit: emulate golang-style multi-returns -- so set `subscribers, flat_digest = await self._get_all_subs`, then `if subscribers is None: return`

Imo it's easier to parse/follow when you don't mix return types